javaslang/src/main/java/javaslang/collection/IndexedSeq.java

```diff
@@ -64,8 +64,7 @@
     @Override
     IndexedSeq<T> distinctBy(Comparator<? super T> comparator);
 
-    @Override
-    <U> IndexedSeq<T> distinctBy(Function<? super T, ? extends U> keyExtractor);
+    @Override <U> IndexedSeq<T> distinctBy(Function<? super T, ? extends U> keyExtractor);
```

Comment:
This is also no option for us :-/ It differs from all other methods of that interface.